### ISSUE TITLE
As for detailed model construction file

### ISSUE THREAD

**Issue by Yi2Zhao** [bolunwang/backdoor#4]
Would you please share the model construction file in here?

**Comment by Shawn-Shan** [comment]
You mean for all the models we have tested in the paper, or only the GTSRB one as in this repo? For the GTSRB model, the model setup and training is at under the injection repo (https://github.com/bolunwang/backdoor/tree/master/injection). The specific model configuration is at https://github.com/bolunwang/backdoor/blob/90bd783c26f1e1ff0574fc64627984d4d390782b/injection/gtsrb_injection_example.py#L55

**Comment by Yi2Zhao** [comment]
Only the GTSRB. I have implemented the model accroding to the paper (Neural Cleanse: Identifying and Mitigating Backdoor Attacks in Neural Networks), of which the model architecture is 6 Con + 2 Dense. But, the test acc on normal gtsrb is too low. So, I want to check what's wrong. Just now, I have check the implementation (load_traffic_sign_model), maybe it's because I used a different optimizer, and I will try it again. Thanks for your help and your time!
